README.md

```diff
@@ -73,6 +73,7 @@ const steps = [
 | `onBeforeClose` | function triggered before close | `func` | `() => { document.body.style.overflowY = 'auto' }`  |  |
 | `onRequestClose` | function triggered to close | `func` |  |  |
 | `prevButton` | prev navigation button text | `string` |  |  |
+| `rounded` | Beautify helper + mask with border-radius (in px) | `number` | `0` |  |
```

Comment:
Seems there's a conflict with missing props added in the last PR (`prevStep`, `nextStep`)